docs/development/arcus.md

```diff
@@ -0,0 +1,290 @@
+
+
+
+# arcus/latest@ultimaker/stable
+
+---
+
+Arcus is one of our own repositories and can be found: https://github.com/Ultimaker/libArcus For each commit to master
```

Comment:
I don't really understand why this is here. THis isn't arcus or cura engine specific, this is for all conan dependencies.